server/db/models/Gig.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function (sequelize, DataTypes) {
+  var Gig = sequelize.define('Gig', {
+    title: DataTypes.STRING,
+    type: DataTypes.ENUM('wedding', 'birthday', 'business'),
+    date: DataTypes.DATE,
+    start_time: DataTypes.TIME,
+    end_time: DataTypes.TIME,
+    complexity: DataTypes.INTEGER,
```

Comment:
Is complexity something we are going to be calculating or just some heuristic inputted by an admin?